actioncable/test/remote_connections_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+require 'stubs/test_server'
+
+class ActionCable::RemoteConnections::BaseTest < ActionCable::TestCase
+  test '#where' do
+    server             = TestServer.new
+    remote_connections = ActionCable::RemoteConnections.new(server)
+    remote_connection  = remote_connections.where(var1: 1, var2: 2, var3: 3)
+    assert_instance_of ActionCable::RemoteConnections::RemoteConnection, remote_connection
+    assert_equal 1, remote_connection.instance_variable_get(:@var1)
```

Comment:
This is still testing the internal state of the object instead of the public API. What do you think about testing the return of `identifiers` method?

Also a test for `disconnect` would be great.